explainer.md

```diff
@@ -194,8 +181,14 @@ To balance the needs of providing for future extensibility while ensuring maximu
 
 ## Considered alternatives
 
->One of the most important things you can do in your design process is to catalog the set of roads not taken. As you iterate on your design, you may find that major choices in your approach or API style will be revisited and enumerating the full space of alternatives can help you apply one (or more) of them later, may serve as a “graveyard” for u-turns in your design, and can give reviewers and potential users confidence that you’ve got your ducks in a row.
+### Stay the course and build machine learning solutions on WebGL/WebGPU
+
+WebGL and WebGPU are Web API abstraction layers to the underlying graphics API, which could be used to implement neural network operations that run on the GPU. Popular Javascript machine learning frameworks such as TensorFlow.JS already uses WebGL and plans to add a WebGPU backend shortly in the future. An alternative to the WebNN proposal is to continue with this architecture and rely on Javascript frameworks implemented with these graphics abstraction layers to address the current and future needs of AI scenarios on the web.
```

Comment:
one nit: s/TensorFlow\.JS/TensorFlow\.js/g

reference: https://www.tensorflow.org/js

Reply:
Fixed. This is quite important. Thanks for catching it.

Reply:
Thanks for fixing it!